lighthouse-core/audits/user-timings.js

```diff
@@ -12,8 +12,8 @@ const UIStrings = {
   /** Descriptive title of a diagnostic audit that provides details on any timestamps generated by the page. User Timing refers to the 'User Timing API', which enables a website to record specific times as 'marks', or spans of time as 'measures'. */
   title: 'User Timing marks and measures',
   /** Description of a Lighthouse audit that tells the user they may want to use the User Timing API to help measure the performance of aspects of their page load and interaction. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Consider instrumenting your app with the User Timing API to create custom, ' +
-      'real-world measurements of key user experiences. ' +
+  description: 'Consider instrumenting your app with the User Timing API to measure your ' +
+      'app\'s real-world performance during key user experiences. ' +
```

Comment:
"instrumenting" plus the "learn more" link seemed sufficient to imply what "custom" was covering before?